service/service_test.go

```diff
@@ -189,7 +202,7 @@ func TestUnitStart(t *testing.T) {
 		svc := createMockService(productMap, mockPayment, mockTransformer, mockDao)
 
 		Convey("Given a message is readily available for the service to consume", func() {
-			svc.Consumer = createMockConsumerWithMessage()
+			svc.Consumer = createMockConsumerWithMessage(paymentResourceID, "")
```

Comment:
Passing empty strings around seems like it can be improved
```
createMockConsumerWithPaymentMessage(paymentResourceID)
createMockConsumerWithRefundMessage(paymentResourceID, refundId)
```